bloom-housing/bloom
# 1901/program questions application

# Pull Request Template

## Issue Overview

This PR addresses #1901


## Description

Adds program questions section into the application management form.

## How Can This Be Tested/Reviewed?

1. Create or use existing listing with defined program questions
2. Add a new application and try multiple program answers combination
3. Make sure that data has been saved correctly
4. Try to edit an application
5. Make sure that edit progress saved proper values

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
